addons/sale/views/sale_views.xml

```diff
@@ -484,16 +482,7 @@
                                         options="{'color_field': 'color'}"
                                         domain="['|', ('company_id', '=', False), ('company_id', '=', parent.company_id)]"
                                     />
-                                    <field
-                                        name="product_uom_qty"
-                                        context="{
-                                            'partner_id': parent.partner_id,
-                                            'quantity': product_uom_qty,
-                                            'pricelist': parent.pricelist_id,
-                                            'uom': product_uom,
-                                            'company_id': parent.company_id
-                                        }"
-                                    />
+                                    <field name="product_uom_qty"/>
```

Comment:
have a look at 1cedcf6abb433beb78675557fabd9ee6e49c504e so you can double-check if this is really unused